src/waltz/quic/fd_quic.c

```diff
@@ -4218,7 +3964,7 @@ fd_quic_conn_tx( fd_quic_t *      quic,
 
     /* did we add any frames? */
 
-    if( !pkt_meta->flags ) {
```

Comment:
If we only add ACK frames then pkt_meta->flags==0.
In this case we should probably free the pkt_meta and treat the outgoing ACK as fire-and-forget.